packages/transactions/src/builders.ts

```diff
@@ -175,18 +178,32 @@ export async function broadcastTransaction(
  */
 export async function broadcastRawTransaction(
   rawTx: Buffer,
-  url: string
+  url: string,
+  attachment?: Buffer
 ): Promise<TxBroadcastResult> {
-  const requestHeaders = {
-    'Content-Type': 'application/octet-stream',
-  };
-
-  const options = {
+  let options = {
     method: 'POST',
-    headers: requestHeaders,
-    body: rawTx,
   };
 
+  if (attachment) {
+    options = _.assign(options, {
```

Comment:
Could you just use the native `Object.assign` right?

Reply:
Yup! I think lodash uses that under the hood if it’s available. I got into the habit of using lodash for this at a previous gig because IE still doesn’t support `Object.assign`, and we needed to support it. Will make this change as I don’t think we’re too concerned with supporting IE

Reply:
I'm not particularly opposed to lodash, but think this imports the entire bundle, versus `import assign from 'lodash-es/assign'`.